### PR TITLE
Update sv.js

### PR DESCRIPTION
**Pull request by prince17dec**
Syntactic bugs. Changed attribut to attribute.

### CODE REVIEW

File: src/locale/sv.js [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+  currency             : 'Fält :attribute måste vara en giltig valuta.',
+  date                 : 'Fält :attribute måste vara ett datum.',
+  date_equals          : 'Fält :attribute måste matcha :date.',
+  email                : 'Fält :attribute måste vara en giltig e-postadress.',
+  in                   : 'Fält valt: attribute måste vara :values.',
+  integer              : 'Fält :attribute måste vara ett heltal.',
+  max                  : 'Fält :attribute får inte överstiga :max:type.',
+  min                  : 'Fält :attribute måste åtminstone vara :min:type.',
+  not_in               : 'Fält vald :attribute får inte vara :values.',
+  not_regex            : 'Fält :attributete får inte matcha det obligatoriska mönstret.',
```
Comment (stuyam) [inline src/locale/sv.js:28]:
```suggestion
  not_regex            : 'Fält :attribute får inte matcha det obligatoriska mönstret.',
```

**Review by prince17dec — COMMENTED** [review]
fixed attributete to attribute

File: src/locale/sv.js [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+  not_in               : 'Fält vald :attribute får inte vara :values.',
+  not_regex            : 'Fält :attribute får inte matcha det obligatoriska mönstret.',
+  numeric              : 'Fält :attribute måste vara ett nummer.',
+  phone                : 'Fält :attribute måste vara ett giltigt telefonnummer.',
+  regex                : 'Fält :attribute måste matcha det obligatoriska mönstret.',
+  required             : 'Fält :attribute krävs.',
+  size                 : 'Fält :attribute måste vara :size:type.',
+  string               : 'Fält :attribute måste vara en sträng.',
+  typeof               : "Fält :attribute är inte den korrekta typen av :type.",
+  url                  : 'Fält :import { api, locale } from '../../index'; måste vara en url.',
```
Comment (stuyam) [inline src/locale/sv.js:36]: This looks to be a typo

**Review by prince17dec — COMMENTED** [review]
fixed typo

File: src/locale/sv.js [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+  before_or_equal      : 'Fält :attribute måste matcha eller vara före :date.',
+  between              : 'Fält :attribute måste vara mellan :min och :max:type.',
+  boolean              : 'Fält :attribute måste vara booleskt.',
+  card_exp             : "Fält :attribute måste vara ett giltigt utgångsdatum.",
+  card_num             : 'Fält :attribute måste vara ett giltigt kreditkortsnummer.',
+  currency             : 'Fält :attribute måste vara en giltig valuta.',
+  date                 : 'Fält :attribute måste vara ett datum.',
+  date_equals          : 'Fält :attribute måste matcha :date.',
+  email                : 'Fält :attribute måste vara en giltig e-postadress.',
+  in                   : 'Fält valt: attribute måste vara :values.',
```
Comment (stuyam) [inline src/locale/sv.js:23]:
```suggestion
  in                   : 'Fält valt :attribute måste vara :values.',
```

**Pull request merged**